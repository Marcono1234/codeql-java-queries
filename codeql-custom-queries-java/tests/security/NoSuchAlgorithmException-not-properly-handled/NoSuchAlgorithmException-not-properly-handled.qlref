./security/NoSuchAlgorithmException-not-properly-handled.ql
