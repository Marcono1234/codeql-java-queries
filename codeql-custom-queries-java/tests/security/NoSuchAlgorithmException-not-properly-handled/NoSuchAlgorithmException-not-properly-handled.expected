| Test.java:14:9:14:42 | catch (...) | Might not properly handle NoSuchAlgorithmException |
| Test.java:23:11:23:68 | catch (...) | Might not properly handle NoSuchAlgorithmException |
| Test.java:33:9:33:27 | catch (...) | Might not properly handle NoSuchAlgorithmException |
